packages/module/src/CloseButton/CloseButton.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import { Button, ButtonProps, ButtonVariant } from '@patternfly/react-core';
 import { CloseIcon } from '@patternfly/react-icons';
-import clsx from 'clsx';
 import { createUseStyles } from 'react-jss';
+import clsx from 'clsx';
 
 const useStyles = createUseStyles({
```

Comment:
@jessiehuff does removing the padding on this component keep the click area large enough to meet a11y standards?

And I think that in general, to keep these components compatible with right-to-left languages, we might want to avoid using `float:right`. I think there is a `float:end` or something like that we should be using. I'm also curious though if there's a PF modifier we could use rather than this custom styling. @mcoker WDYT?

Reply:
@nicolethoen good call on the RTL support. There are logical values for `float` - "inline-end" (used instead of "right" - what you'd use here) and "inline-start" (used instead of "left").

> I'm also curious though if there's a PF modifier we could use rather than this custom styling

We don't have a global modifier for this. We do have [float utility classes](https://www.patternfly.org/utility-classes/float) though, and that utility stylesheet is lightweight (https://unpkg.com/browse/@patternfly/patternfly@latest/utilities/Float/float.css) if that's an option, though it's worth noting that our utility classes are not using logical values when they probably should. I opened an issue for that here https://github.com/patternfly/patternfly/issues/6273.

Reply:
Based on WCAG guidelines, the clickable area should be 44 x 44 pixels. It looks like removing the padding makes it 48 x 36 so we should probably at least increase the height to 44 pixels.

Reply:
i think we are going to need a designer to step in at some point to commit a review once or twice a months of new components.  with the changes in v6 I can see this giving us headaches.  Fortunately we are small enough right now w can fix stuff.